okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -136,6 +141,34 @@ public Level getLevel() {
     return level;
   }
 
+  /** Change the limitation of request body logs size. */
+  public HttpLoggingInterceptor setRequestBodyLogMax(long requestBodyLogMax) {
```

Comment:
Is it simpler to have a single limit for logging? How much value is there in having two with separate values?

I guess on sending you can get it over and done with and not hold onto the memory, higher chance you might received a slow response from the server and hold things in memory.

What's your thinking on this?

Reply:
I don't have a strong opinion to separate values. It's just for convenience.
Simpler is also better for me, so I change it to the single value 👍